src/main/java/com/sarality/db/query/AggregateMeasure.java

```diff
@@ -0,0 +1,28 @@
+package com.sarality.db.query;
+
+import com.sarality.db.Column;
+
+/**
+ * TODO @Satya what is this for?
+ *
+ * @author Satya@ () on 16/11/16
+ */
+
+public class AggregateMeasure {
```

Comment:
explained the raison d'etre for this class in the SimpleQueryBuilder. In a nutshell, column can be non unique and hashMap wouldnt work then for us.